madgraph/iolibs/export_v4.py

```diff
@@ -8000,6 +8013,7 @@ def create_couplings_part(self, nb_file, data, dp=True, mp=False, vec=False):
               parameter  (PI=3.141592653589793d0)
               parameter  (ZERO=0d0)
               include 'input.inc'
+              include '../vector.inc'
               include 'coupl.inc'""")
```

Comment:
same (again one which is less clear if we need it or not)

Reply:
same as above, vector.inc is needed by coupl.inc